core/src/mindustry/world/blocks/defense/turrets/Turret.java

```diff
@@ -59,6 +59,14 @@
     public boolean alternate = false;
     public boolean targetAir = true;
     public boolean targetGround = true;
+    /** Charging related stuff*/
```

Comment:
This won't end well on the java documentation. It'll show like this:

`float chargeTime Charging related stuff`

I assume you are confused by the "Currently used for artillery only" which is for the `minRange` field. You can see it [here](https://mindustrygame.github.io/docs/mindustry/world/blocks/defense/turrets/Turret.html#minRange).